Create host-guide.md (#731)
jupyter/jupyter: docs/source/community/host-guide.md

*from the Jupyter Security group if needed*)
- [ ] Make sure you have a stable internet
connection, a good quality microphone, and a quiet environment.
- [ ] Check the [meeting agenda](insert agenda)
  - *The agenda is a collaboratively created document made by the participants.*
    *Ensure a section for today's meeting is created, with today's date and a table*
    *for participants to record their names (which can be copied/pasted from lower*
    *entries). The agenda will also be used as meeting minutes.*

&#x1F517; *Links to copy/paste into the meeting chat:*

- Weekly Meeting Notes: [Insert your team’s notes here]
- Jupyter Code of Conduct: https://jupyter.org/governance/conduct/code_of_conduct.html
- [Subproject Name]Team Compass: 

### &#x1F399; Start the Meeting! 

> Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
> [host name] and I’ll be your host today.

> > This is a place to for contributors to connect with each other

Reviewer: Should we clarify what 'contributors' mean here? It sounds like it's for code contributors and maintainers only, which sounds exclusionary. The meetings are open to public, so we should encourage anyone to bring up issues, questions and suggestions

info, advice and best practices for conducting engaging and effective
meetings.

## &#x23F0; Basic Meeting Event Info 

You can check the [Jupyter Community Meetings calendar](https://jupyter.org/community#calendar) for the exact time of each meeting in your time zone.

Each meeting should have a collaborative public agenda document (for example on HackMD). The agenda link should be included in the event’s description in the shared Google Calendar.

## &#x1F3AF; Meeting Purpose/Goals

Meetings are a way to connect members of our 
community. All participants should feel welcome and be heard. In line with that:

Meetings should be...

- Positive and friendly
- Welcoming and open to people from diverse backgrounds
- Newcomers in particular should be welcomed and encouraged to
  chime in, discuss, participate, or to simply observe if that is their preference.

## &#x1F4AC; Topics
Each meeting should have a list of topics that are generally discussed. Contributors can propose topics before or during the meeting.

(If using this document as a template, replace this section with a list of topics that are discussed at your meeting.)

> &#x1F4AA; Host Responsibilities - Martha

Reviewer 2: ```suggestion

## &#x1F4AA; Host Responsibilities
```

Promotes this to an H2, removes the owner's name

  - *Do time checks/pause + move on to hit all items when there are time concerns*
- [ ] Guiding discussion:
  - *Read chats aloud for participants + the recording*
- [ ] Maintaining order / Making sure everyone has a chance to speak:
  - *Interject when someone has their hand raised*
- [ ] Mention/link to the Jupyter Code of Conduct and how everyone is bound to it (including the host)

Partner with another contributor (a meeting facilitator) to do the following:

- [ ] Log in as the “Project Jupyter” host account for meeting management perks (removing spammers, muting and recording, etc.), [request access](mailto:security@ipython.org) from the Jupyter Security Team.
- [ ] Record the meeting, if that is intended. The host starts and stops the recording. These are posted to YouTube.
- [ ] Remove unwelcome recording bots:
  - Per the Jupyter community page, participants
  should not record meetings on their own. The host should ask participants
  that look like AI recording bots:
  - To identify themselves and state whether they're recording
  - If they get no response from the account, the host should assume
    it's recording, and remove it from the meeting.

> ## &#x270f; Suggested Meeting Script and Checklist 
> ### &#x23F3; Prior to meeting start

Reviewer 2: ```suggestion

## &#x270f; Suggested Meeting Script and Checklist 

### &#x23F3; Before the Meeting Starts
```

Copy edit, Title Case

> - [ ] Log in to Zoom from the Project Jupyter account [(*request access*](mailto:security@ipython.org). Note this is necessary in order to record and manage meeting participants.  Alternatively, provided that someone who has host privileges logs onto the call they can grant host privileges in real time at the start of the meeting. 

Reviewer 2: ```suggestion
- [ ] Log in to Zoom from the Project Jupyter account (request access by emailing [security@ipython.org](mailto:security@ipython.org)). This is necessary in order to record and manage meeting participants.  Alternatively, provided that someone who has host privileges logs onto the call they can grant host privileges in real time at the start of the meeting. 
```

This makes the email address more visible without hovering or clicking.

*from the Jupyter Security group if needed*)
- [ ] Make sure you have a stable internet
connection, a good quality microphone, and a quiet environment.
- [ ] Check the [meeting agenda](insert agenda)
  - *The agenda is a collaboratively created document made by the participants.*
    *Ensure a section for today's meeting is created, with today's date and a table*
    *for participants to record their names (which can be copied/pasted from lower*
    *entries). The agenda will also be used as meeting minutes.*

&#x1F517; *Links to copy/paste into the meeting chat:*

- Weekly Meeting Notes: [Insert your team’s notes here]
- Jupyter Code of Conduct: https://jupyter.org/governance/conduct/code_of_conduct.html
- [Subproject Name]Team Compass: 

> ### &#x1F399; Start the meeting! 

Reviewer 2: ```suggestion
### &#x1F399; Start the Meeting! 
```

Title case

> ```
> Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
> [host name] and I’ll be your host today.
> This is a place to for contributors to connect with each other
> and the community about [TOPIC_HERE] topics. A special highlight and welcome to all first time participants! We want all newcomers to feel
> welcome, we invite you to join in on discussions, introduce yourself,
> or add items to the agenda.
> 
> Please keep in mind that this call will be recorded and posted to YouTube for the community to view. This call is a part of the Jupyter community, therefore we follow Jupyter Code of Conduct, which you can read about at jupyter.org/conduct[Meeting host insert link in the meeting minutes template]
> 
> Before we begin, I want to share how topics make it on to the agenda. Presenters have submitted what they want to discuss or share ahead of time. We’ll have a little time for discussion with each one.
> ```

Reviewer 2: ```suggestion
> Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
> [host name] and I’ll be your host today.
> This is a place to for contributors to connect with each other
> and the community about [TOPIC_HERE] topics. A special highlight and welcome to all first time > participants! We want all newcomers to feel
> welcome, we invite you to join in on discussions, introduce yourself,
> or add items to the agenda.
> 
> Please keep in mind that this call will be recorded and posted to YouTube for the community to view. This call is a part of the Jupyter community, therefore we follow Jupyter Code of Conduct, which you can read about at jupyter.org/conduct[Meeting host insert link in the meeting minutes template]
> 
> Before we begin, I want to share how topics make it on to the agenda. Presenters have submitted what they want to discuss or share ahead of time. We’ll have a little time for discussion with each one.
```

The `> ` syntax makes the text appear as a blockquote, not as formatted source code.

> [Prepare to start the recording]

Reviewer 2: ```suggestion
Prepare to start the recording:

```

Consistency with other example scripts

> ```
> Before we start the recording, does anyone have anything they'd like
> to say off the record? (Say "We will now begin record/The recording has ended" before starting/stopping
> recording :)
> ```

Reviewer 2: ```suggestion
> Before we start the recording, does anyone have anything they'd like
> to say off the record? (Make an announcement immediately before starting and after ending the recording.)
```

Styles as blockquote, copy edit

### &#x1F3BA; Closing message 

> ```
> Thank you to everyone for joining! I’m happy to have you all as a part
> of the Jupyter community.
> If you have any feedback, please direct it at [platform/link - could be Discourse, SSC rep email, Subproject email, Team Compass]. We’d love to hear what
> you think about the call or how to improve future meetings.
> Our next call will be [day, month]. Check out the [Jupyter community calendar](https://jupyter.org/community#calendar)to see what other types of meetings are happening in between.
> ```

Reviewer 2: ```suggestion
> Thank you to everyone for joining! I’m happy to have you all as a part
> of the Jupyter community.
> If you have any feedback, please let us know at [platform/link - could be Discourse, SSC rep email, Subproject email, Team Compass]. We’d love to hear what
> you think about the call or how to improve future meetings.
> Our next call will be [day, month]. Check out the [Jupyter community calendar](https://jupyter.org/community#calendar)to see what other types of meetings are happening.
```

Blockquote style, copy edits

> For Community Call, delete if not: 
> ```
> If you have any interest in sharing in future TOPIC_HERE  calls, please do! It doesn’t have to be a big share, a polished share, or a technical share, just show off what you do with Jupyter that excites you. Submit topics here [platform/link].
> If you’re interested in hosting a TOPIC_HERE  call, feel free to reach out. It would be great to get different people interacting with the community.
> ```

Reviewer 2: ```suggestion
The following message is recommended for Community Calls or similar events that engage with the broader contributor community:

> If you have any interest in sharing in future [title] calls, please do! It doesn’t have to be a big share, a polished share, or a technical share, just show off what you do with Jupyter that excites you. Submit topics here [platform/link].
> If you’re interested in hosting a [title] call, feel free to reach out. It would be great to get different people interacting with the community.
```

Copy edits, rewrites intro, blockquote styling

> [After the recording ends]
> ```
> If you have anything you would like to add off the record, please feel free to discuss that now.
> ```

Reviewer 2: ```suggestion
After the recording ends:

> If you have anything you would like to add off the record, please feel free to discuss that now.
```

Copy edit, style edit
